src/platform/plugins/shared/inspector/public/index.ts

```diff
@@ -7,10 +7,6 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-// TODO: https://github.com/elastic/kibana/issues/109900
```

Comment:
I think this comment is irrelevant since we don't export with a `*` in this plugin.